app/helpers/application_helper.rb

```diff
@@ -1,4 +1,13 @@
 # frozen_string_literal: true
 
 module ApplicationHelper
+  # ページごとの完全なタイトルを返します。
+  def full_title(page_title = '')
+    base_title = 'Ruby on Rails Tutorial Sample App'
+    if page_title.empty?
+      base_title
+    else
+      "#{page_title} | #{base_title}"
+    end
```

Comment:
早期リターン(「リーダブルコード」にも出てくるかも?)を使うとネストを減らせて便利です。
```suggestion
    return base_title if page_title.empty?
    
    "#{page_title} | #{base_title}"
```